src/AWS.Deploy.Common/Recipes/AvailableRuleItem.cs

```diff
@@ -1,37 +0,0 @@
-﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-// SPDX-License-Identifier: Apache-2.0
-
-using System.Collections.Generic;
-
-namespace AWS.Deploy.Common.Recipes
-{
-    /// <summary>
-    /// Container for the types of rules that can be checked.
-    /// </summary>
-    public class AvailableRuleItem
```

Comment:
where did this go?

Reply:
This class was not used at all. There were no references for it. I also ran the integ tests and they passed.